### DISCUSSION TITLE
Windows ARM

### DISCUSSION THREAD

**Discussion by me1299** [ente-io/ente#3512] [Enhancements]
Could you please compile a Windows client for ARM systems? The installer fails with the error `This program does not support the version of Windows your computer is running.`

When downloading the ZIP and running the executable it complains about certificates being incorrect.
![image](https://github.com/user-attachments/assets/79777324-8707-4e7b-a626-3dbd9beb311b)

I'm using a new Surface Pro with a Snapdragon ARM processor.

**Comment by NathanNunemaker** [comment]
I am having the same issue. I am daily driving an AMR laptop to test it for work and it would be awesome if Ente Auth would work (:

**Comment by me1299** [comment]
You can make a workaround with Google Chrome's 'install page as app' function.

![image](https://github.com/user-attachments/assets/4c4137e5-3535-4635-a126-a3ec2242e4e8)
